code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -3,7 +3,7 @@
 //TODO: Make these simple_animals
 
 #define MIN_IMPREGNATION_TIME 100 //time it takes to impregnate someone
-#define MAX_IMPREGNATION_TIME 150
+#define MAX_IMPREGNATION_TIME 150 // WHY DOES THIS NOT SPECIFY UNITS
```

Comment:
I agree but please don't place comments on tg files just for colour commentary, this would be a silly reason to get a merge conflict :p

Reply:
> I agree but please don't place comments on tg files just for colour commentary, this would be a silly reason to get a merge conflict :p

oops! I think I meant to remove that and forgot 😅